package.json

```
@@ -10,7 +10,7 @@
     "lib"
   ],
   "peerDependencies": {
-    "eslint": ">=4.19.1"
+    "eslint": "^6.0.0 || ^7.0.0"
```

I believe we already support eslint v8. It is tested on CI.

```suggestion
    "eslint": "^6.0.0 || ^7.0.0 || ^8.0.0"
```

---

I made a mistake. eslint v8 is not tested yet 😓